contracts/tfi-factory/src/mock_querier.rs

```diff
@@ -94,18 +119,47 @@ impl WasmMockQuerier {
                         .with_commission(pair_info.commission),
                     )))
                 } else {
-                    panic!("DO NOT ENTER HERE")
+                    SystemResult::Err(SystemError::UnsupportedRequest {
+                        kind: format!("key {:#?} not supported", key),
```

Comment:
Some leftover, or is pretty print actually helpful here? :thinking:

Reply:
Not really useful, just better than a panic.